### ISSUE TITLE
Remove regex from assets.yml

### ISSUE THREAD

**Issue by mcwumbly** [pivotal-cf/kiln#21]
Recently, @rowanjacobs did some work to simplify the `assets.yml` file by removing the requirement to include a regex in this pull request: https://github.com/pivotal-cf/kiln/pull/17:

Unfortunately, the solution we landed on there ended up causing `kiln fetch` to delete releases under certain circumstances, as reported here: https://github.com/pivotal-cf/kiln/issues/20

We'd still like to remove the regex, but will have to take a different approach, either:

1. include a "subdirectory" instead (as first suggested and implemented in the pull request) _or_
2. wait until we stop storing duplicate compiled releases in separate subfolders.

**Comment by rowanjacobs** [comment]
Nah, I've pretty much stopped caring now that I'm not a kiln developer or user. Put all the ugly regexes in there you want. I recommend starting with [this puzzle](https://regexcrossword.com/playerpuzzles/8cbea27f-c4c5-4d11-a509-6a622ba01107) @njbennett
